TokenSale.sol

```diff
@@ -48,14 +57,16 @@ contract TokenSaleInterface {
 
 
 contract TokenSale is TokenSaleInterface, Token {
-    function TokenSale(uint _minValue, uint _closingTime, address _privateSale) {
+    function TokenSale(uint _minValue, uint _closingTime, address _privateSale) 
```

Comment:
This goes agains the style guide. In this case, if it's only 1 char above the limit I would not bother (till we have automated tools that do it for us).

But normally the way you would do this depends on style. One way to break it is:

```
    function TokenSale(
        uint _minValue,
        uint _closingTime,
        address _privateSale
) {
```